root/mbid/NotFound.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import NotFound from '../components/NotFound';
+
+const MbidNotFound = ({isGuid, mbid}: {+isGuid: boolean, +mbid: string}) => (
+  <NotFound title={isGuid ? l('MBID Not Found') : l('Invalid MBID')}>
+    <p>
+      {isGuid ? (
+        exp.l(
+          `No MusicBrainz {entity_doc|entities} match the {mbid_doc|MBID}
+           {mbid}. Either it’s incorrect, it was for an entity that has since
+           been deleted, or it is an ID for something else than an entity
+           (for example, a {rel_type_table|relationship type}).`,
+          {
+            entity_doc: '/doc/MusicBrainz_Entity',
+            mbid: mbid,
+            mbid_doc: '/doc/MusicBrainz_Identifier',
+            rel_type_table: '/relationships',
+          },
+        )
+      ) : (
+        texp.l(
+          '{mbid} is not a valid MBID.',
```

Comment:
* The same doc link as above would be nice to have.
* This sentence doesn’t make sense from `/mbid`, `/mbid?q=invisible_man`, `/mbid//invisible_man`, or `/mbid/reosarevok`.

Reply:
It does make sense for `/mbid/reosarevok` (`reosarevok` is not a valid MBID!) but I added a separate option if no mbid is passed ("No MBID selected.").